app.rb

```diff
@@ -0,0 +1,324 @@
+require_relative './student'
```

Comment:
[REQUIRED] I am getting an error when I run your code and this is preventing me from being able to see what you have done

|Your Code|
|------|
|<img width="1266" alt="Screenshot 2022-10-01 at 17 23 13" src="https://user-images.githubusercontent.com/62938735/193418997-6e7db362-b4be-42c9-be60-216e760981db.png">|

Reply:
Corrections made to the appropriate files